snapcraft.yaml

```diff
@@ -20,5 +20,9 @@ parts:
     source: .
     plugin: go
     go-importpath: github.com/OpenBazaar/openbazaar-go
+    go-packages: [github.com/OpenBazaar/openbazaar-go]
```

Comment:
This is needed because of #432, but it doesn't hurt. Otherwise it will try to install all the packages, when only this one is required.